crux-uberjar/test/crux/uberjar_test.clj

```diff
@@ -5,13 +5,14 @@
             [clojure.java.shell :as sh]
             [crux.fixtures :as f]
             [clojure.tools.logging :as log]
-            [crux.io :as cio]))
+            [crux.io :as cio])
+  (:import (java.io File)))
 
-(def working-directory
-  (-> (io/as-file (io/resource "crux/uberjar_test.clj"))
-      .getParentFile
-      .getParentFile
-      .getParentFile))
+(def ^File working-directory
+  (.. (io/as-file (io/resource "crux/uberjar_test.clj"))
```

Comment:
First, parens around all threaded function calls. Second, I never use `..`. YMMV. There are other instances like this in this PR, but won't comment on each one of them.